config/config.exs

```diff
@@ -34,4 +34,9 @@ config :canary,
   repo: CoursePlanner.Repo,
   unauthorized_handler: {CoursePlanner.Helper, :handle_unauthorized}
 
+config :course_planner, CoursePlanner.Scheduler,
+  jobs: [
+    {"0 18 * * *", {CoursePlanner.Notifications, :send_all_notifications, []}}
```

Comment:
There's a problem in heroku free tier with crons. For now I didn't find a good solution, but the job does not run if the instance is sleeping.